### ISSUE TITLE
No padding around card

### ISSUE THREAD

**Issue by sedestrian** [joogps/SlideOverCard#10]
Hi, I have iOS 15 and iPhone 13 Pro, the card that appears on my screen touches the edges of the screen and has no padding on left, right and bottom.

**Comment by Amnell** [comment]
@sedestrian Without code it is quite hard to help. The issue is probably due to some modifiers that's propagating to the Card.

**Comment by startcode-io** [comment]
Same error

**Comment by Scott1803** [comment]
Were you able to solve this issue? Even in the demo preview, there is no padding. Is it possible to add padding between the popup and the edges of the screen?

**Comment by mickeyl** [comment]
In my local copy I'm using the following patch: 

```swift
diff --git a/Sources/SlideOverCard/SlideOverCard.swift b/Sources/SlideOverCard/SlideOverCard.swift
index 772e279..8c597ee 100644
--- a/Sources/SlideOverCard/SlideOverCard.swift
+++ b/Sources/SlideOverCard/SlideOverCard.swift
@@ -54,6 +54,7 @@ public struct SlideOverCard<Content: View>: View {
                 Group {
                     if #available(iOS 14.0, *) {
                         container
+                            .padding(8)
                             .ignoresSafeArea(.container, edges: .bottom)
                     } else {
                         container
```

to make it look more like the "original".

You may want to check though whether the changes that have landed upstream earlier this month solve your problem as well by introducing inner- and outer padding options.
